enterprise/server/backends/userdb/userdb.go

```diff
@@ -361,6 +361,20 @@ func (d *UserDB) CreateUserAPIKey(ctx context.Context, groupID, label string, ca
 	return createdKey, nil
 }
 
+func (d *UserDB) DeleteUserGitHubToken(ctx context.Context) error {
+	u, err := perms.AuthenticatedUser(ctx, d.env)
+	if err != nil {
+		return err
+	}
+	if u.GetUserID() == "" {
+		return status.FailedPreconditionError("user ID must not be empty")
+	}
+	return d.h.DB(ctx).Exec(
+		`UPDATE Users SET github_token = "" WHERE user_id = ?`,
```

Comment:
To support postgres, we are going to use more standard sql quoting which uses single quotes for string values and double quotes for identifiers. Let's start using single quotes for strings for any new queries we add.

Reply:
Done - and will keep this in mind